cmd/stchaind/faucet.go

```diff
@@ -196,7 +309,30 @@ func getFirstAccAddressFromGenesis(cdc *codec.Codec, genesisFilePath string) (ac
 func doTransfer(cliCtx context.CLIContext, txBldr authtypes.TxBuilder, to sdk.AccAddress, from sdk.AccAddress, coin sdk.Coin) error {
 	//// build and sign the transaction, then broadcast to Tendermint
 	msg := bank.NewMsgSend(from, to, sdk.Coins{coin})
-	cliCtx.BroadcastMode = "block"
+	cliCtx.BroadcastMode = "async"
```

Comment:
why change to async? if it's async, it doesn't guarantee the tx is successfully sent

Reply:
I tried both async and sync, both of them work for multiple requests. And I agree that if we want to have a return of the tx, sync mode should be better.